activiti-core/activiti-spring-boot-starter/src/test/java/org/activiti/spring/boot/process/ProcessRuntimeIT.java

```diff
@@ -824,6 +822,20 @@ public void should_handleBigDecimalAndDoubleAndLocalDateTimeVariables() {
                 tuple("doubleVar", doubleValue, "double"),
                 tuple("localDateTimeVar", localDateTime, "localDateTime"),
                 tuple("localDateVar", localDate, "localDate")
-                );
+            );
+    }
+
+    @Test
+    public void should_processAdminOnlyProcessDefinitionsFromLatestVersionRetrieved() {
```

Comment:
In addition to this test, I think you need one similar to [this one](https://github.com/Activiti/Activiti/blob/develop/activiti-core/activiti-spring-boot-starter/src/test/java/org/activiti/spring/boot/ApplicationUpgradeIT.java#L101-L118) covering the method `getProcessDefinition` from admin API.

Reply:
done, thanks for the suggestion!